filesystems/xfs/xfstests/README.md

```diff
@@ -0,0 +1,21 @@
+# xfstests test suite
+Wrapper for xfstests test suite, see http://git.kernel.org/?p=fs/xfs/xfstests-dev.git;a=summary \
+Test Maintainer: [Xiong Murphy Zhou](mailto:xzhou@redhat.com)
+
+## How to run it
+
+### Configure which test cases to run, e.g.
+```bash
+$ export TEST_PARAM_FSTYPE=xfs
+$ export TEST_PARAM_RUNTESTS=generic/001
+```
+
+### Dependencies
+Dependencies will be automatically installed with dep-install.sh when you execute \
```

Comment:
Could you add a sentence before this one saying to refer to the top-level README.md for common dependencies? People are going to land here from URLs in the reports and get confused, if they don't follow that and try to run the test.